kw

```diff
@@ -50,17 +62,27 @@ if [ -f "${KW_BASE_DIR}/src/lib/kwlib.sh" ]; then
   KW_ETC_DIR="${KW_BASE_DIR}/etc"
   KW_PLUGINS_DIR="${KW_LIB_DIR}/plugins"
   # KW_CACHE_DIR # use default cache folder
-fi
-##END-DEV-MODE##
 
-# Export external variables required by kworkflow
-export KWORKFLOW
+  # include IO utilities
+  include "${KW_LIB_DIR}/lib/kwio.sh"
 
-#INJECT_CODE_TRACING_SETUP
+  # Print a notice to the developers so he knows some envvars have been changed
+  # We redirect to stderror because this output would make tests fails if the developer is testing
+  say "[INFO]: Running kw in developer mode. Envvars set to the following:
```

Comment:
Besides changing the reference `developer mode` to the new name, maybe break the line before `Envvars set to (...)`. Also, we could be a little more verbose with and use `kw environment variables set to the following:`.

Reply:
alright, I'll make it